Gemfile

```diff
@@ -23,6 +23,10 @@ group :development do
   gem 'github-markup'
 
   # for visual tests
-  gem 'sinatra'
+  if RUBY_VERSION < '2.2.0'
+    gem 'sinatra', '~> 1.4.8'
+  else
```

Comment:
How about making this explicit with:
```ruby
gem 'sinatra', RUBY_VERSION >= "2.2.0" ? "~> 2.0" : "~> 1.7.0"
```

Reply:
It's already explicit in the gemspec, so I don't see the benefit of repeating it here. We only need the exception.

Reply:
> It's already explicit in the gemspec

Am I looking at a different version here?
https://github.com/jneen/rouge/blob/e957c740ca85b3a96f2cea1e8e1eafbf9cb2abc4/rouge.gemspec#L5-L25

Reply:
My mistake. You're absolutely right.

Reply:
Though that then raises the question why not specify versions for all the other dependencies in Gemfile. Again, I'm just trying to get the build working again, not changing the versioning policy.

Reply:
And my intention was to avoid mentioning `gem 'sinatra'` twice here. :smiley:

Ignore me.